components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsMessageTable/src/EIAnalyticsMessageTable.jsx

```diff
@@ -191,11 +179,9 @@ class EIAnalyticsMessageTable extends Widget {
                     .subscribeWidget(
                         this.props.id, this.handleStats, dataProviderConf
                     );
-
-
             })
             .catch((error) => {
-                // todo: Handle error
+                console.log(error);
```

Comment:
This is not the correct way to handle errors. Either show a meaningful message to the user, or put a detailed error log.

Reply:
Fixed at b233d5335fc34cc9a03e24a4944cd39d0f6c3677